discord/ext/commands/core.py

```diff
@@ -1304,18 +1323,19 @@ async def only_me(ctx):
 
     Parameters
     -----------
-    predicate: Callable[[:class:`Context`], :class:`bool`]
-        The predicate to check if the command should be invoked.
+    *predicates: Callable[[:class:`Context`], :class:`bool`]
+        An argument list of predicates to check if the command should be invoked.
     """
 
     def decorator(func):
-        if isinstance(func, Command):
-            func.checks.append(predicate)
-        else:
-            if not hasattr(func, '__commands_checks__'):
-                func.__commands_checks__ = []
+        for predicate in predicates:
+            if isinstance(func, Command):
+                func.checks.append(predicate)
+            else:
+                if not hasattr(func, '__commands_checks__'):
+                    func.__commands_checks__ = []
 
-            func.__commands_checks__.append(predicate)
+                func.__commands_checks__.append(predicate)
```

Comment:
```py
if isinstance(func, Command):
    func.checks.extend(predicates)
else:
    if not hasattr(func, '__commands_checks__'):
        func.__commands_checks__ = []
    func.__commands_checks__.extend(predicates)
```
This removes a level of nesting and avoids repeated conditional calls

Reply:
I see, great!